deep-learning/_posts/2018-11-30-NeuroNet.md

```diff
@@ -29,7 +30,7 @@ individual software package.
 # Introduction
 
 The disparity of results in brain tissue segmentation in neuroimaging tools
-such as FSL[FSL], SPM[SPM], and MALPEM[MALPEM] puts neuroscientists before
+such as FSL[^1], SPM[^2], and MALPEM[MALPEM] puts neuroscientists before
```

Comment:
MALPEM => ^3

Reply:
That's having a sharp eye 🦅 . Thanks ! 0ad89cf fixes it.